packages/api/__tests__/index.js

```diff
@@ -35,6 +35,7 @@ let server;
 const serverUrl = `http://localhost:${PORT}`;
 
 beforeAll((done) => {
+  jest.setTimeout(20000);
```

Comment:
If you added this due to Travis build errors, I think that is more because there is no mongo instance to which the app running on Travis is connecting to, rather than some DB calls really needing > 5 seconds to complete.